res/values/strings.xml

```diff
@@ -209,22 +209,22 @@
     <string name="lockscreen_access_pattern_cell_added">Pattern cell added</string>
     <string name="lockscreen_access_pattern_detected">Pattern detected</string>
     <string name="lockscreen_access_pattern_passwd_forget">Forgot password</string>
-    <string name="lockscreen_access_pattern_failure">Failed 5 times, please wait for 30 seconds before trying</string>
+    <string name="lockscreen_access_pattern_failure">Failed 5 times, please wait 30 seconds before trying</string>
     <string name="lockscreen_access_pattern_failure_left_try_seconds">You can retry %d seconds later</string>
     <string name="lockscreen_access_pattern_failure_left_try_times">Incorrect pattern, you can try another %d times</string>
-    <string name="lockscreen_access_pattern_failure_not_long_enough">Linking points not long enough, pelase try again</string>
+    <string name="lockscreen_access_pattern_failure_not_long_enough">Linking points not long enough, please try again</string>
     <string name="lockpattern_retry_button_text">Retry</string>
     <string name="lockpattern_continue_button_text">Next</string>
     <string name="lockpattern_confirm_button_text">OK</string>
     <string name="lockpattern_recording_intro_header">Please draw Gesture Lock Pattern</string>
     <string name="lockpattern_settings_help_how_to_record">Learn how to draw Gesture Lock Pattern</string>
-    <string name="lockpattern_recording_incorrect_too_short">Link at least 4 points, try again!</string>
+    <string name="lockpattern_recording_incorrect_too_short">Swipe across at least 4 points, try again!</string>
     <string name="lockpattern_pattern_entered_header">Pattern saved</string>
     <string name="lockpattern_need_to_draw">Please draw Gesture Lock pattern</string>
     <string name="lockpattern_need_to_confirm">Please confirm your pattern</string>
-    <string name="lockpattern_need_to_unlock_wrong">Pattern not match, try again!</string>
-    <string name="lockpattern_pattern_confirmed_header">Confirm to save your pattern</string>
-    <string name="lockpattern_recording_inprogress">Move away your firgure after drawing"</string>
+    <string name="lockpattern_need_to_unlock_wrong">Pattern didn't match. Try again.</string>
```

Comment:
I pulled your commit and run the project, the IDE complained an error, here "didn\`t" should be changed like this "didn\\`t"